app/assets/javascripts/surveys.js

```diff
@@ -34,6 +35,26 @@
         frequency: 50,
         startTime: new Date("July 26, 2016").getTime(),
         endTime: new Date("July 27, 2016 23:59:59").getTime()
+      },
+      {
+        url: 'https://www.surveymonkey.com/s/2MRDLTW',
+        identifier: 'user_satisfaction_survey',
+        template: TEMPLATE,
+        frequency: 10,
+        activeWhen: function() {
+          function breadcrumbMatches() {
+            var text = $('#global-breadcrumb').text() || "";
+            return (/Education and learning/.test(text) || /Schools and education/.test(text));
```

Comment:
Im not wild about this mechanism of detecting "education content". I'm going to push back and see if the logic below in `sectionMatches()` will match enough to make this a worthwhile test.

OTOH, this specific callback is only temporary and will be removed in a couple of weeks after the test has run.